Website/src/items/powershell-scripts/EdgeWebsite/DAM Image Inventory.yml

```diff
@@ -0,0 +1,110 @@
+Parent: "6053403f-2ebd-4546-bc1c-869bbb66c320"
+Template: "dd22f1b3-bd87-4db2-9e7d-f7a496888d43"
+Path: /sitecore/system/Modules/PowerShell/Script Library/SPE/Reporting/Content Reports/Reports/EdgeWebsite/DAM Image Inventory
+SharedFields:
+- ID: "06d5295c-ed2f-4a54-9bf2-26228d113318"
+  Hint: __Icon
+  Value: Office/32x32/revert_database.png
+- ID: "b1a94ff0-6897-47c0-9c51-aa6acb80b1f0"
+  Hint: Script
+  Value: |
+    $ContentHubHostname = 'mint.stylelabs.io'
+    $customList = New-Object System.Collections.Generic.List[PSObject]
+    $allItems = Get-ChildItem -Path 'master://sitecore/content/' -Recurse
+    $allItems | ForEach-Object {
+        foreach ($field in $_.Fields) {
+            if ($field.Type -eq "Image" ) {
+                $fieldValue = $_.Fields[$field.Name].Value
+                $htmlDocument = New-Object -TypeName HtmlAgilityPack.HtmlDocument
+                $htmlDocument.LoadHtml($fieldValue)
+                foreach ($x in $htmlDocument.DocumentNode.SelectNodes("//image")) { 
+                    $src = $x.Attributes["src"].Value
+                    if ( [string]::IsNullOrEmpty($src) ) {
+                        $src = $x.Attributes["mediaid"].Value
+                    }
+                    $formatted = '<p style="background-color:Salmon; padding:5px;">' + $src + '</p>'
+                    if($src -like "*$ContentHubHostname*"){
+                        $formatted = '<p style="background-color:PaleGreen; padding:5px;">' + $src + '</p>'
+                    }
+                    $newObject = New-Object PSObject @{ ItemPath = $_.FullPath; ItemName = $_.DisplayName; FieldName = $field.Name; FieldType = 'Image'; Value = $formatted; }
+                    $customList.Add($newObject)
+                }
+            }
+            if ($field.Type -eq "Rich Text" ) {
+                $fieldValue = $_.Fields[$field.Name].Value
+                $htmlDocument = New-Object -TypeName HtmlAgilityPack.HtmlDocument
+                $htmlDocument.LoadHtml($fieldValue)
+                foreach ($x in $htmlDocument.DocumentNode.SelectNodes("//img")) { 
+                    $src = $x.Attributes["src"].Value
+                    $formatted = '<p style="background-color:Salmon; padding:5px;">' + $src + '</p>'
+                    if($src -like "*$ContentHubHostname*"){
+                        $formatted = '<p style="background-color:PaleGreen; padding:5px;">' + $src + '</p>'
+                    }
+                    $newObject = New-Object PSObject @{ ItemPath = $_.FullPath; ItemName = $_.DisplayName; FieldName = $field.Name; FieldType = 'Rich Text'; Value = $formatted; }
+                    $customList.Add($newObject)
+                }
+            }
+        }
+    }
+    $reportProps = @{
+        Title           = "Media Item Inventory"
+        InfoTitle       = "All the Media Items..."
+        InfoDescription = "Total entries found: $($customList.length)"
+        PageSize        = 25
```

Comment:
At first glance, I did not see there were multiple pages. It would be nice to write something like "(Make sure to check all pages)" in the InfoDescription after the number of entries found.